doc/governance/index.rst

```diff
@@ -0,0 +1,116 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+  * Decision-making is consensus-driven by those who show up.
+  * Leadership roles are defined primary by responsibility, not prestige or seniority.
+    * It is normal and healthy for these roles to be passed on to others
+  * The role of developers is ultimately to serve the users.
+
```

Comment:
Should we spell out that none of these roles are compensated?

Though at the same time we may want to somehow guarantee travel compensation and conference fee waivers for the CEC or CSC, perhaps funded by the foundation? (Without this, travel costs to leadership meetups could exclude certain candidates from participating in Ceph governance in future).